### ISSUE TITLE
Avoid additional tracking cookies

### ISSUE THREAD

**Issue by serpent213** [lalabuy948/PhoenixAnalytics#29]
Would be nice to have the option to use only the default Phoenix session's `_phoenixapp_key` cookie for tracking purposes and avoid setting any additional ones.

Lovely project!  💠

**Comment by lalabuy948** [comment]
Hi @serpent213 , 

Thank you for suggestion, I was slightly busy in last month, will try to take a look what can be done.

**Comment by franc** [comment]
agreed, would be great if we could configure it to not need additional cookie
